tests/kas_with_instance.spec.ts

```diff
@@ -173,3 +183,35 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_6acl.py test_kafka_create_consumer_group_and_check_dashboard
+test('create consumer group and check dashboard', async ({ page }) => {
+  const instanceLinkSelector = page.getByText(testInstanceName);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Connection').click();
+
+  const broker = await page.locator('[aria-label="Bootstrap server"]').getAttribute('value');
```

Comment:
Could you use `.inputValue()` here instead? https://playwright.dev/docs/api/class-locator#locator-input-value

Reply:
changed to `const broker = await page.locator('[aria-label="Bootstrap server"]').inputValue();`

Reply:
Can I resolve this conversation? @mmusil